src/PowerShellEditorServices/Services/PowerShellContext/Session/PowerShellVersionDetails.cs

```diff
@@ -134,7 +134,7 @@ public static PowerShellVersionDetails GetVersionDetails(Runspace runspace, ILog
                         versionString = powerShellVersion.ToString();
                     }
 
-                    var arch = PowerShellContextService.ExecuteScriptAndGetItem<string>("$env:PROCESSOR_ARCHITECTURE", runspace);
+                    var arch = PowerShellContextService.ExecuteScriptAndGetItem<string>("$env:PROCESSOR_ARCHITECTURE", runspace, useLocalScope: true);
```

Comment:
So does it turn out we need this to be in script after all?

Reply:
yep... only one way to get the PROCESSOR_ARCHITECTURE over remoting...

Reply:
I figured it was remoting